src/uwtools/config.py

```diff
@@ -0,0 +1,198 @@
+'''
+This file contains the Config file and its subclasses for a variety of
+dicatable file types.
+'''
+
+import abc
+import collections
+import configparser
+
+import f90nml
```

Comment:
So, if other "dictable" filetypes are added, all those will be imported here and not in the class that defines it?

Reply:
Let's hope we don't need that!